lib/routes/account.js

```diff
@@ -66,7 +66,8 @@ module.exports = function (
               name: isA.string().max(255).required(),
               type: isA.string().max(16).required(),
               pushCallback: isA.string().uri({ scheme: 'https' }).max(255).optional().allow(''),
-              pushPublicKey: isA.string().length(64).regex(HEX_STRING).optional().allow('')
+              // We're not yet ready to store pubkey values, don't let clients submit them.
```

Comment:
Perhaps it should read "don't let clients submit non-empty strings", but depending on client behaviour, you're right that rejecting anything with a `pushPublicKey` item in the request is probably cleaner